src/ngraph/op/mpi_bcast.hpp

```diff
@@ -0,0 +1,37 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <memory>
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        class MPI_Broadcast : public Op
```

Comment:
The class should be named MPIBroadcast without the underscore.
Today it is implemented with MPI but will it always be that way. If we change the implementation to use some other message passing protocol then the class name would be wrong. Use a more general name than `MPI`, something like BroadcastDistributed so it sorts with Broadcast yet is unique.
Make sure you change the header file names to match too.

Reply:
Renamed the op name DistBroadcast. All the header file names are changed too.

Reply:
Agree, `BroadcastDistributed` would be a better name .

Reply:
@jianyinglang have you get a chance to see this ⬆️ ?

Reply:
Okay, I will change the name to `BroadcastDistributed`